packer/ansible/roles/pm2/files/rackhd.yml

```diff
@@ -0,0 +1,16 @@
+apps:
```

Comment:
Suggest to change the file name to avoid the confusion with "packer/ansible/rackhd_local.yml", something like "rackhd-pm2-config.yml" will be better.